### PR TITLE
Mapped parts of ScreenHandler

### PR DESCRIPTION
**Pull request by Genau6502**
I have mapped a couple of methods in ScreenHandler and added names for a few variables. It's my first time contributing so would like to know if there is anything I should change.
method_30010 -> removeStack
Renamed a few auto-generated names in the methods in ScreenHandler
Mapped getStack()
Mapped totalSlots()

### CODE REVIEW

File: mappings/net/minecraft/screen/ScreenHandler.mapping [hunk 1]
```diff
@@ -48,9 +51,11 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 	METHOD method_7593 onSlotClick (IILnet/minecraft/class_1713;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1799;
 		COMMENT Performs a slot click. This can behave in many different ways depending mainly on the action type.
 		COMMENT @return The stack that was clicked on before anything changed, used mostly for verifying that the client and server are in sync
+		ARG 1 slotNum
```
Comment (jaskarth) [inline mappings/net/minecraft/screen/ScreenHandler.mapping]: what does `Num` mean here? Is it the slot id or the slot count?

File: mappings/net/minecraft/screen/ScreenHandler.mapping [hunk 2]
```diff
@@ -33,8 +33,11 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 		ARG 0 context
 		ARG 1 player
 		ARG 2 block
-	METHOD method_30010 (IILnet/minecraft/class_1713;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1799;
+	METHOD method_30010 removeStack (IILnet/minecraft/class_1713;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1799;
+		ARG 1 stackNum
```
Comment (jaskarth) [inline mappings/net/minecraft/screen/ScreenHandler.mapping]: ditto

**Review by jaskarth — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/util/collection/DefaultedList.mapping [hunk 3]
```diff
@@ -7,7 +7,7 @@ CLASS net/minecraft/class_2371 net/minecraft/util/collection/DefaultedList
 	METHOD add (ILjava/lang/Object;)V
 		ARG 1 value
 		ARG 2 element
-	METHOD get (I)Ljava/lang/Object;
+	METHOD get getStack (I)Ljava/lang/Object;
```
Comment (Juuxel) [inline mappings/net/minecraft/util/collection/DefaultedList.mapping]: No. This is an override of a `List` method, and the name should **not** be touched. (There's also nothing about this class that's specific to slots.)

File: mappings/net/minecraft/util/collection/DefaultedList.mapping [hunk 4]
```diff
@@ -18,6 +18,7 @@ CLASS net/minecraft/class_2371 net/minecraft/util/collection/DefaultedList
 		ARG 1 defaultValue
 	METHOD remove (I)Ljava/lang/Object;
 		ARG 1 index
-	METHOD set (ILjava/lang/Object;)Ljava/lang/Object;
+	METHOD set setSlots (ILjava/lang/Object;)Ljava/lang/Object;
```
Comment (Juuxel) [inline mappings/net/minecraft/util/collection/DefaultedList.mapping]: Same here, this is from `List`.

File: mappings/net/minecraft/util/collection/DefaultedList.mapping [hunk 5]
```diff
@@ -18,6 +18,7 @@ CLASS net/minecraft/class_2371 net/minecraft/util/collection/DefaultedList
...
 		ARG 1 index
 		ARG 2 element
+	METHOD size totalSlots ()I
```
Comment (Juuxel) [inline mappings/net/minecraft/util/collection/DefaultedList.mapping]: And this one is from `Collection`.

**Review by Juuxel — CHANGES_REQUESTED** [review]

**Comment by Genau6502** [comment]
Let me know if there's anything else you would like me to clear up/fix. Thanks for your feedback :)

File: mappings/net/minecraft/screen/ScreenHandler.mapping [hunk 6]
```diff
@@ -48,9 +51,11 @@ CLASS net/minecraft/class_1703 net/minecraft/screen/ScreenHandler
 	METHOD method_7593 onSlotClick (IILnet/minecraft/class_1713;Lnet/minecraft/class_1657;)Lnet/minecraft/class_1799;
 		COMMENT Performs a slot click. This can behave in many different ways depending mainly on the action type.
 		COMMENT @return The stack that was clicked on before anything changed, used mostly for verifying that the client and server are in sync
+		ARG 1 slotid
```
Comment (Juuxel) [inline mappings/net/minecraft/screen/ScreenHandler.mapping]: Hmm, is this the slot ID (`Slot.id`) or its index in `ScreenHandler.slots`?

**Review by Gegy — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**